src/scripts/main.js

```diff
@@ -1 +1,81 @@
 'use strict';
+
+let leftClicked = false;
+let rightClicked = false;
+
+const firstPromise = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(Error('First promise was rejected'));
+  }, 3000);
+
+  document.body.addEventListener('click', e => {
+    leftClicked = true;
+    document.body.removeEventListener('click', this);
```

Comment:
you could add once listener instead of removing it manually
https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#parameters

see options -> once

Reply:
thanks!